### ISSUE TITLE
Using keyboard keys the StatusBar is not accessible.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51977]
**Related Issues:** https://github.com/Microsoft/vscode/issues/51975, https://github.com/Microsoft/vscode/issues/51981

**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

Additional Details:
Keyboard,MAS2.1.1,MAS2.4.3,MAS2.4.7

**Repro Steps:**
1.Navigate to "Status Bar" at bottom of the screen after login.
2.Try to navigate "Status Bar" through the keyboard key such as "TAB".
3.Observed that,Using tab key "Status Bar" is not accessible.

**Actual:**
Using mouse only user can access the item list of "StatusBar".Using keyboard keys such as "tab,arrow keys" the statusBar is not accessible to user. Also because of this keyboard focus is not visible on the any item of statusbar.

**Expected:**
Without using mouse user should be able to access the item of "StatusBar".Using "Tab and arrow keys" user should be able to access all the item list of "Status Bar".Keyboard focus should be visible on each selected item.but because of not able to navigate using keyboard keyboard focus is also loss.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/fixit-details.aspx?fixitid=326](url)
OR
[https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx](url)

**User Impact:**
Only keyboard using user will not able to access the "StatusBar"

**MAS Reference:**
MAS2.1.1- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](url)
MAS2.4.3- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}](url)
MAS2.4.7- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}](url)

**Attachment For Reference:**
[[Accessibility]A11y_VSCode_StatusBar_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2105291/Accessibility.A11y_VSCode_StatusBar_Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by bpasero** [comment]
@AccessibilityTestingTeam-TCS @auchenberg we explicitly decided to not make the status bar keyboard accessible to prevent that a user ends up in many many items down there until going on. All of the action items in the status bar can be accessed via the command palette too. I am not sure we should change this.

**Comment by AccessibilityTestingTeam-TCS** [comment]
As per C and E accessibility testing the application should be accessible by the each and every person.The bug is related to "StatusBar is accessible by the mouse only" here the bug is logged for considering the "Keyboard Only" user. Keyboard only user will not able to access the status bar. And if the functionality is like this then please let me know more.

**Comment by isidorn** [comment]
Using tab key the status bar should not be accessible that is a design decision which I agree with. We do not just blindly follow standards and we believe this we lead to the best user experience possible.
@bpasero as a workaround could you introduce a command palette actions: "Focus status bar"

**Comment by bpasero** [comment]
@isidorn not sure, you can only focus something if it has a `tabindex`, so if we enter this mode, would we add this to each status bar entry and then remove it when the last element lost focus? I would like to better understand why we have to do this because all of the executable actions in the status bar should also be triggered through the command palette. If it is about reading the text there to a screen reader, we could maybe have an action that does just that.

**Comment by isidorn** [comment]
@bpasero makes sense. Yeah let's wait and see what is the actual benefit of doing this.
